examples/rules/prevent-password-plain-text-jfrog.json

```diff
@@ -0,0 +1,49 @@
+{
+    "description": "Prevent use of password as plain text",
+    "failureMessage": "a password key is used without obfuscation. We recommend using github secrets manager or gitlab-vault instead",
+    "uniqueId": 13,
+    "enabledByDefault": true,
+    "inCodeImplementation": false,
+    "schema": {
+      "definitions": {
+          "passwordPattern": {
+              "type": "string",
+              "pattern": "\\${{.*}}( *|$)|DYNAMIC_VALUE"
+          },
+          "lookup": {
+              "properties": {
+                  "configuration": {
+                      "type": "object",
+                      "patternProperties": {
+                          ".*password.*": {
+                              "$ref": "#/definitions/passwordPattern"
+                          },
+                          "inputResources": {
+                              "type": "array",
+                              "items": {
+                                  "type": "object",
+                                  "patternProperties": {
+                                      ".*password.*": {
+                                          "$ref": "#/definitions/passwordPattern"
+                                      }
+                                  }
+                              }
+                          }
+                      }
+                  }
+              }
+          }
+      },
+      "allOf": [
+          {
+              "$ref": "#/definitions/lookup"
+          }
+      ],
+      "additionalProperties": {
+          "$ref": "#"
+      },
+      "items": {
+          "$ref": "#"
+      }
+  }
+}
```

Comment:
add newline at the end of the file
I recommend using this extension: samverschueren.final-newline